auth-server/src/services/reservationListener.ts

```diff
@@ -0,0 +1,88 @@
+import MqttService from '../services/mqtt';
+import fs from 'fs';
+
+const url = process.env.SERVER_MQTT_URL || 'mqtt://localhost';
+const topic = process.env.RESERVATION_TOPIC || 'rooms/+/reservations/#';
+const publicKeyPath = process.env.PUBLIC_KEY_PATH || 'pubkey.pem';
+
+let publicKey: string;
+try {
+  publicKey = fs.readFileSync(publicKeyPath).toString();
```

Comment:
Remove the try/catch block, i think we want to fail fast and crash the server if publicKeyPath is missing.

Reply:
If there is some specific reason, please add a comment